app/controllers/admins_controller.rb

```diff
@@ -29,4 +29,11 @@ def email_instructors
   def github_integration
     @github_integration = GithubIntegration.check_github_authorization
   end
+
+  action_auth_level :clear_cache, :administrator
+  def clear_cache
+    Rails.cache.cleanup
+    flash[:success] = "Cache Cleared"
+    redirect_to root_path
```

Comment:
For a better UX, I would suggest changing this to `redirect_back(fallback_location: root_path)`

This redirects the user to their previous page if possible, and falls back to the root path otherwise.

Ref: https://apidock.com/rails/v6.0.0/ActionController/Redirecting/redirect_back

Reply:
Fixed.